connection-service/azuredeploy.json

```diff
@@ -308,6 +278,12 @@
           },
           "CSUniqueSuffix": {
             "value": "[parameters('CSUniqueSuffix')]"
+          },
+          "enableExternalAccess": {
+            "value": "[bool(parameters('enableExternalAccess'))]"
+          },
+          "enableRadiusMfa": {
```

Comment:
We could just make this part of CAMDeploymentParameters but I'm not sure it's worth it...